cmd/busyambassador/main.go

```diff
@@ -13,11 +13,11 @@ import (
 	"strings"
 
 	// 1st-party libs
+	"github.com/datawire/ambassador-agent/cmd/agent"
```

Comment:
So do you expect Emissary to consume the agent as an image, or as a Go library?

Reply:
At this point it does both, but I think we want to consume as an image. I don't understand this package enough to know if I could just rip out this import